### PR TITLE
Update docs/forms page content

### PR DESCRIPTION
**Pull request by kwm14**
## Done
- Updated copy ✏️ 
- Updated demo example to showcase live example 💯 
- Check content matches copy doc [here](https://docs.google.com/document/d/1n_FNUUnBt3Mnka_c6w3lp2ov84IpC8p0Na6t7THP16s/edit#heading=h.2iegp7r10no)

## QA
- Pull code
- Run /docs > ./run
  - Check http://0.0.0.0:8104/en/base/forms
- Pull code
- Run ./run
- Review many many many updated examples
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/form/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/checkboxes/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/disabled-input/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/fieldset/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/input/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/labels/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/select-multiple/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/radio-buttons/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/selects/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/textarea/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/forms/form-help-text/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/forms/form-inline/ 
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/forms/forms-required/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/forms/form-stacked/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/forms/form-validation/

## Details
- Fixes https://github.com/ubuntudesign/design-vanilla-framework/issues/352

## Screenshots
Screenshot 2019-04-01 at 14 16 33
Screenshot 2019-04-01 at 14 18 28

### CODE REVIEW

File: docs/en/base/forms.md [hunk 1]
```diff
@@ -91,16 +99,22 @@ By applying the class `.p-form--stacked` and wrapping any form control in `.p-fo
 View examples of form stacked patterns
 </a>
 
-### Disabled state
+### Disabled
+
+Adding the `[disabled="disabled"]` attribute to an input will prevent user interaction.
```
Comment (deadlight) [inline docs/en/base/forms.md]:
```suggestion
Adding the `disabled` attribute to an input will prevent user interaction.
```
The disabled attribute doesn't need a key and value

File: docs/en/base/forms.md [hunk 2]
```diff
@@ -34,15 +38,19 @@ View example of an input element
 
 Note: The attribute `readonly` disables the input but it still retains a default cursor.
 
-### Checkbox and radio button
+### Checkbox
 
-Use checkboxes and radio buttons to select one or more options.
+Use checkboxes to select one or more options, default checkboxes can appear in two states: selected and disabled, or unselected.
```
Comment (deadlight) [inline docs/en/base/forms.md]: Aren't there 4 states? Both selected, unselected have enabled and disabled elements

File: docs/en/base/forms.md [hunk 3]
```diff
@@ -34,15 +38,19 @@ View example of an input element
...
 
 <a href="https://vanilla-framework.github.io/vanilla-framework/examples/base/forms/checkboxes/"
     class="js-example">
 View example of the base checkboxes
 </a>
 
+### Radio button
+
+Use radio buttons to make a single selection, our radio buttons can appear in two states: selected and disabled, or unselected.
```
Comment (deadlight) [inline docs/en/base/forms.md]: Same comment as above: 4 states

**Comment by kwm14** [comment]
@anthonydillon updated changes based on @deadlight feedback. Ready for re-review 👍

File: docs/en/base/forms.md (hunk 2 above)
Comment (anthonydillon) [reply docs/en/base/forms.md]: This is valid. There are more then two states as Karl says.

File: docs/en/base/forms.md (hunk 3 above)
Comment (anthonydillon) [reply docs/en/base/forms.md]: Same here

**Comment by anthonydillon** [comment]
Looking good, just need to update the content on the states of select boxes.

**Comment by kwm14** [comment]
@anthonydillon updated 😃

**Pull request merged**